projects/canopy/src/lib/table/table-cell/table-cell.component.ts

```diff
@@ -101,6 +101,10 @@ export class LgTableCellComponent {
     return this._showLabel;
   }
 
+  @HostBinding('class.lg-table-cell--stacked')
+  @Input()
+  stackSm: true;
```

Comment:
should this read `stackSm = true;`

Reply:
🤦   Actually it needs to be this:

```
@Input()
stackSm = false;
```

Fixed now!

Reply:
ah yes that matches the notes :-)

Reply:
Nice spot, thanks Aaron 👍